net.sourceforge.fretsonfire.appdata.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.sourceforge.fretsonfire</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <name>Frets On Fire</name>
+  <summary>Frets on Fire is a game of musical skill and fast fingers. The aim of the game is to play guitar with the keyboard as accurately as possible.</summary>
+  <description>
+    <p>
+      Frets on Fire is a game of musical skill and fast fingers. The aim of the
+      game is to play guitar with the keyboard as accurately as possible.
+    </p>
+    <p>Features:</p>
+    <ul>
+      <li>Unique inverted keyboard gameplay style</li>
+      <li>Support for guitar controllers and generic joysticks</li>
+      <li>Includes a song editor for making your own tunes</li>
+      <li>Compete with others on the World Charts</li>
+      <li>Hundreds of songs composed by the community</li>
+      <li>Supports importing Guitar Hero™ I and Guitar Hero™ II songs</li>
+      <li>Multiplatform; runs on Windows™, Linux®, Mac OS X™ and FreeBSD®</li>
+      <li>Open source, full Python source code available</li>
+    </ul>
+    <p>Requirements:</p>
+    <ul>
+      <li>128 MB of RAM</li>
+      <li>A fairly fast OpenGL graphics card with decent drivers</li>
+      <li>Windows: Direct X compatible sound card</li>
```

Comment:
Could you remove the non-Linux requirements from the appdata? If we're patching it in then we don't have to be cross platform

Reply:
done